tomcat-gcp-lib/src/main/java/com/google/cloud/runtimes/tomcat/logging/HttpLabels.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.runtimes.tomcat.logging;
+
+class HttpLabels {
+  public static final String REQUEST_SIZE = "/request/size";
+  public static final String RESPONSE_SIZE = "/response/size";
```

Comment:
Could you rename this field `/http/response/size' to match the specification in the [Stackdriver Documentation](https://cloud.google.com/trace/docs/viewing-details#timeline) ?

Reply:
Done!